test/integration/apiary.js

```diff
@@ -0,0 +1,199 @@
+// Tests CAP-38 endpoints against the mock API server:
+// https://ammmock.docs.apiary.io/
+
+// All endpoints from here are tested:
+// https://docs.google.com/document/d/1pXL8kr1a2vfYSap9T67R-g72B_WWbaE1YsLMa04OgoU/edit
+const MOCK_SERVER = "http://private-anon-a06e1b25a0-ammmock.apiary-mock.com";
+
+describe("tests the /liquidity_pools endpoint", function() {
+  const lpId = "0569b19c75d7ecadce50501fffad6fe8ba4652455df9e1cc96dc408141124dd5";
+
+  it("GET /", function(done) {
+    let server = new StellarSdk.Server(MOCK_SERVER, {allowHttp: true});
+
+    chai.request(MOCK_SERVER)
+      .get("/liquidity_pools")
+      .end(function(err, res) {
+        if (err != null) done(err);
+        expect(res.body).not.to.be.null;
+
+        server
+          .liquidityPools()
+          .call()
+          .then(resp => {
+            expect(resp.records).to.deep.equal(res.body._embedded.records);
+            done();
+          })
+          .catch(err => done(err));
+      });
+  });
+
+  it('GET /<id>', function(done) {
+    let server = new StellarSdk.Server(MOCK_SERVER, {allowHttp: true});
+
+    chai.request(MOCK_SERVER)
+      .get(`/liquidity_pools/${lpId}`)
+      .end(function(err, res) {
+        if (err != null) done(err);
+        expect(res.body).not.to.be.null;
+
+        server
+          .liquidityPools()
+          .liquidityPoolId(lpId)
+          .call()
+          .then(resp => {
```

Comment:
❔  Why are you using arrow functions here but regular functions in the `it`? I prefer arrow functions all around if we can do that.

Reply:
I like arrow functions for smaller blocks of code. That aside, mocha (the test library) says [not to use](https://mochajs.org/#arrow-functions) arrow functions for `describe`/`it` because they modify the test context (or some other JS nonsense subtlety I don't fully understand).

Reply:
TIL! Thanks for explaining that 😄